src/intl/en/page-layer-2.json

```
@@ -122,7 +122,7 @@
   "scaling-layer-1-with-shard-chains": "Scaling layer 1 with shard chains",
   "understanding-rollup-economics-from-first-principals": "Understanding rollup economics from first principals",
   "arbitrum-description": "Arbitrum is an Optimistic Rollup that aims to feel exactly like interacting with Ethereum, but with transactions costing a fraction of what they do on L1.",
-  "optimism-description": "Optimistic Ethereum is an EVM-equivalent Optimistic Rollup chain. It aims to be fast, simple, and secure.",
+  "optimism-description": "Optimism is a fast, simple, and secure EVM-equivalent Optimistic Rollup. It scales Ethereum's tech while also scaling its values through retroactive public goods funding.",
```

@corwintines was there a reason we wanted this shorter? just making sure this character length doesn't break any designs.


Also, we should stick to site conventions here... always "optimistic rollup" unless the former begins the sentence. I notice we're doing this everywhere... 😕 will fix in a separate PR.

```suggestion
  "optimism-description": "Optimism is a fast, simple, and secure EVM-equivalent optimistic rollup. It scales Ethereum's tech while also scaling its values through retroactive public goods funding.",
```

---

I'm wondering what the value of '_It scales Ethereum's tech while also scaling its values through retroactive public goods funding_' is to users. It's kinda vague and doesn't explain Optimism's unique selling point to those who don't already know about it... any thoughts @corwintines? No strong opinion on removing it, just don't think it achieves a lot here.

---

Pretty indifferent. It definitely adds an Optimism flair to the description which I can see as a reason for keeping it. No strong opinion on removing it though.

---

Also, description length is fine. It was one of the shorter ones anyways. Now its comparable to Boba and Arbitrum.